azavea/temperate
# Remove custom UI slider styling

## Overview

Fixes the chart slider not rendering in the proper range for the chart by removing the custom styling overriding the [noUiSlider](https://github.com/leongersen/noUiSlider) library.


### Demo

![image](https://user-images.githubusercontent.com/960264/63896139-32385580-c9bf-11e9-9149-1105779de20e.png)


### Notes

The styles were overriding those for noUiSlider:
https://github.com/leongersen/noUiSlider/blob/master/distribute/nouislider.css

@azavea/temperate-design might want to reapply some custom styling for the current version of the library.


## Testing Instructions

 * Go to a chart under 'Climate data' in http://localhost:4210/indicators
 * Click 'See more dates'
 * The year range slider control should function as expected



Fixes #1264.
